samples/samples-js-v4/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
```

Comment:
I don't see a local.settings.json, you need to `git add -f` that directly since it's ignored by default.